kyaml/openapi/openapi.go

```diff
@@ -50,6 +52,13 @@ type openapiData struct {
 	schemaInit bool
 }
 
+type format string
+
+const (
+	JSON  format = "json"
```

Comment:
Looking at the implementation below, it seems we can handle json or yaml when the format is `json`.
Maybe we can call it JsonOrYaml

Reply:
Good catch! Changed to JsonOrYaml